code/modules/admin/verbs/playsound.dm

```diff
@@ -57,10 +70,16 @@ GLOBAL_LIST_EMPTY(sounds_cache)
 	var/list/sounds = file2list("sound/serversound_list.txt")
 	sounds += GLOB.sounds_cache
 
-	var/melody = input("Select a sound from the server to play", "Server sound list") as null|anything in sounds
+	var/sound/melody = input("Select a sound from the server to play", "Server sound list") as null|anything in sounds
 	if(!melody)	return
 
-	play_sound(melody)
+	log_admin("[key_name(src)] played server sound [melody]")
+	message_admins("[key_name_admin(src)] played server sound [melody]", 1)
+
+	for(var/mob/M in GLOB.player_list)
+		if(prefs.sound & SOUND_LOBBY)
+			SEND_SOUND(src, sound(melody, repeat = 0, wait = 0, volume = 85 * prefs.get_channel_volume(CHANNEL_LOBBYMUSIC), channel = CHANNEL_LOBBYMUSIC))
```

Comment:
[`repeat`](http://www.byond.com/docs/ref/info.html#/sound/var/repeat) and [`wait`](http://www.byond.com/docs/ref/info.html#/sound/var/wait) both default to `0`, so you shouldn't need them here.
```suggestion
			SEND_SOUND(src, sound(melody, volume = 85 * prefs.get_channel_volume(CHANNEL_LOBBYMUSIC), channel = CHANNEL_LOBBYMUSIC))
```